frontend/src/store/index.js

```diff
@@ -25,6 +25,144 @@ Vue.use(Vuex)
 export const store = new Vuex.Store({
 
   state: {
+    addIndividualITAsteps: [
```

Comment:
What's the reasoning for storing all of these components in the store?

Reply:
hmm your comment has a single highlighted line but you ask about "all of these components".  If you mean the data pertaining to the state of the examCaptureTab, they are in the store because the multiple components, some of which don't have a direct parent-child relationship, need to access and manipulate their values and so using the store was a good option.  I had originally tried to confine them to local state but it just wasn't working.

Reply:
I mean for the questions that belong to that step in the form. I can understand pushing `currentStep` into the store, but this looks to be just straight config.